extjsdk/src/main/java/io/vantiq/extjsdk/ExtensionServiceMessage.java

```diff
@@ -40,6 +40,12 @@
     public static final String PROPERTY_MESSAGE_HEADERS = "messageHeaders";
 
     // Set in parameters to allow reconnect after network half-open situation
+    // This is sent on any CONNECT_EXTENSION messages.  It is used when a connector
+    // "re-connects" to the server to verify that it's the same connector connecting in
+    // (otherwise, the reconnect is rejected).  So, while it's send on all CONNECT... calls,
+    // it's use is in only in the reconnect case (that is, usurping an existing connection).
+    // The name's a bit confusing, but CONNECT_SECRET sounds too much like something you need
+    // to connect (which, under normal circumstances, you don't).
```

Comment:
```suggestion
    // This is sent on any CONNECT_EXTENSION messages.  It is used when a connector
    // "re-connects" to the server to verify that it's the same connector connecting in
    // (otherwise, the reconnect is rejected).  So, while it's sent on all CONNECT... calls,
    // its use is only in the reconnect case (that is, usurping an existing connection).
    // The name's a bit confusing, but CONNECT_SECRET sounds too much like something you need
    // to connect (which, under normal circumstances, you don't).
```